mlprimitives/jsons/mlprimitives.candidates.dsp.SpectralMask.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "mlprimitives.candidates.dsp.SpectralMask",
+    "author": "Rodrigo Diez <rjdiez@mit.edu>",
+    "documentation": "",
+    "description": "Anomalies detection in satellite telemetry data using a spectral mask",
+    "modalities": [],
+    "primitive": "mlprimitives.candidates.dsp.SpectralMask",
+    "fit": {
+        "method": "fit",        
+        "args": [
+            {
+                "name": "X",
+                "description": "Training vector",
+                "type": "ndarray"
+            }
+        ]
+    },
+

```

Comment:
Not critical, but it would be great if you can remove these blank lines.

Reply:
ok

Reply:
so, no blank lines in the whole file, right?

Reply:
right! exactly what it would like if you used a pretty-print tool on the json